src/apps/demo-app/src/app/media-query-status/BUILD.bazel

```diff
@@ -0,0 +1,50 @@
+load("//tools:defaults.bzl", "ng_module", "ts_library", "ts_web_test_suite")
+
+package(default_visibility = ["//visibility:public"])
+
+ng_module(
+    name = "media-query-status",
+    srcs = glob(["**/*.ts"], exclude = ["**/*.spec.ts"]),
+    assets = glob(["**/*.html"]),
+    tsconfig = "//src/apps/demo-app:tsconfig.json",
+    deps = [
+        "//src/lib:layout",
+        "@angular//packages/core",
+        "@angular//packages/forms",
+        "@angular//packages/router",
+        "@layoutdeps//@types",
+    ],
+)
+
+ts_library(
+    name = "test_lib",
+    testonly = 1,
+    srcs = glob(["*.spec.ts"]),
+    deps = [
+        "media-query-status",
+        "@angular//packages/core",
+        "@angular//packages/core/testing",
+        "@angular//packages/platform-browser",
+        "@angular//packages/platform-browser-dynamic/testing",
+        "@angular//packages/platform-browser/animations",
+    ],
+)
+
+ts_web_test_suite(
+    name = "test",
+    srcs = [],
+    # do not sort
```

Comment:
Explain why this cannot be sorted. Similarly in other files.

Otherwise you should consider using a macro for this if it's used too often across multiple packages.

Reply:
Moved into the defaults.bzl macro. Great suggestion!